src/Commands/Integration.ts

```diff
@@ -88,12 +91,12 @@ export class Integration implements CommandModule {
           .register(RabbitMQBusOptions, {
             useValue: {
               exchange: 'EventBus',
-              clientQueue: `integration:${args.accessKey}`,
+              clientQueue: `agent:${args.accessKey}`,
```

Comment:
Why did you rename the client queue?

```suggestion
              clientQueue: `integration:${args.accessKey}`,
```

Reply:
because of there is no suitable name in the table with permissions. (I found only `^agent:.{1,}$`)

Reply:
If you update your branch and run migration, you will be surprised)